src/main/java/com/ensao/gi5/lint/rules/IfElseRule.java

```diff
@@ -0,0 +1,40 @@
+package com.ensao.gi5.lint.rules;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ensao.gi5.lint.constantes.Constantes;
+import com.ensao.gi5.lint.rules.violations.Violation;
+import com.ensao.gi5.lint.visitor.IfElseVisitors;
+import com.ensao.gi5.lint.wrapper.CompilationUnitWrapper;
+import com.ensao.gi5.lint.wrapper.StatementWrapper;
+
+public class IfElseRule  extends Rule{
+	
+	public IfElseRule() {
+		super(Constantes.LINT_REG_018, Level.LOW);
+	}
+	
+	@Override
+	public void apply(CompilationUnitWrapper compilationUnit) {
+		final List<StatementWrapper> ifElse = new ArrayList<>();
+		compilationUnit.accept(new IfElseVisitors(), ifElse);	
+		
+		ifElse.forEach(p->{
+				if(!p.getStatement().contains("{") && !p.getStatement().contains("}")) {
```

Comment:
```java 
 if(condition) for(....) {
}
``` 
will say the `if` statement has `{ }`, which is false.

Use visitor pattern to avoid String format of the statements